core/node/main/builder.cpp

```diff
@@ -433,6 +440,22 @@ namespace fc::node {
         genesis_timestamp,
         std::chrono::seconds(kEpochDurationSeconds));
 
+    o.storage_market_ipld = o.ipld_leveldb;
```

Comment:
Please, pay attention that ipld over leveldb is reused.

Reply:
Is this always LeveldbDatastore? Could storage market work with InMemoryDatastore?

Reply:
Yes, it is an IpldPtr interface and leveldb or in-memory implementation can stand for it. I discussed usage of ipld and raw files with @turuslan and there is no need to have persistent IPLD in storage market client since we can store and operate raw CAR files. I'm going to rewrite this part.

Reply:
Ok

Reply:
IPLD was removed from Storage Market Client and PieceIO.